RecoLocalTracker/SiPixelRecHits/python/PixelCPEGeneric_cfi.py

```diff
@@ -54,3 +54,7 @@
 (phase2_tracker & (phase2_squarePixels | phase2_3DPixels)).toModify(PixelCPEGenericESProducer,
                                                                     NoTemplateErrorsWhenNoTrkAngles = False # use genErrors in the seeding step (when no track angles are available)
                                                                     )
+from Configuration.ProcessModifiers.alpaka_cff import alpaka
```

Comment:
@lguzzi 

this is misplaced in this file as it overwrites the settings from the reconstruction with the HLT menu one, see the log of [wf  12434.402](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-e4bd8b/41250/runTheMatrixGPU-results/12434.402_TTbar_14TeV+2023_Patatrack_PixelOnlyAlpaka/step3_TTbar_14TeV+2023_Patatrack_PixelOnlyAlpaka.log)

```
----- Begin Fatal Exception 03-Sep-2024 14:58:48 CEST-----------------------
An exception of category 'EventSetupConflict' occurred while
   [0] Calling beginJob
Exception Message:
two EventSetup Producers want to deliver type="PixelCPEFastParamsHostPhase2" label="PixelCPEFastParamsPhase2"
 from record PixelCPEFastParamsRecord. The two providers are 
1) type="PixelCPEFastParamsESProducerAlpakaPhase2@alpaka" label="hltESPPixelCPEFastParamsPhase2"
2) type="PixelCPEFastParamsESProducerAlpakaPhase2@alpaka" label="pixelCPEFastParamsESProducerAlpakaPhase2"
Please either
   remove one of these Producers
   or find a way of configuring one of them so it does not deliver this data
   or use an es_prefer statement in the configuration to choose one.
----- End Fatal Exception -------------------------------------------------
```

please make this change self-contained in the phase-2 HLT menu.

Reply:
done in 78974e0384070734cc0e58c72f1b636004558fa2